diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -234,6 +234,24 @@ namespace diff_drive_controller{
       return false;
     }
 
+    // Export parameters to param server if not available
+    if (lookup_wheel_separation)
+    {
+      controller_nh.setParam("wheel_separation", wheel_separation_);
```

Comment:
You are right. 
Maybe they could be set under a sub-namespace, say `/mobile_base_controller/runtime/those_parameters`.
Any better idea to export those params ?